ogr/ogrsf_frmts/hana/ogrhanautils.cpp

```diff
@@ -0,0 +1,227 @@
+/******************************************************************************
+ *
+ * Project:  SAP HANA Spatial Driver
+ * Purpose:  OGRHanaUtils class implementation
+ * Author:   Maxim Rylov
+ *
+ ******************************************************************************
+ * Copyright (c) 2020, SAP SE
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ ****************************************************************************/
+
+#include "ogrhanautils.h"
+
+#include <algorithm>
+
+CPL_CVSID("$Id$")
+
+namespace hana_utils {
+
+CPLString StringFormat(const char* format, ...)
```

Comment:
just use CPLString().Printf(const char* format, ...)

Reply:
Done in 0a714561f809f9bb023f94eaae32471ea7acc238.